src/react/components/_finances/TransactionsCard/BillingItem/ChargeItem.scss

```diff
@@ -1,5 +1,6 @@
-.ChargeItem__due {
+.TableColumn__due.ChargeItem__due {
   flex-direction: column;
   align-items: flex-end;
   color: #474747;
```

Comment:
@developish can we change the hex color hardcoding to variables?

Reply:
Yes, this one I want to leave for the moment until we have a conversation with the designers